drone_localization/core/services/refinement.py

```
@@ -88,20 +86,16 @@ def calculate_position(
+        with Image.open(drone_image) as img:
             x = transformation["UAV"](img).unsqueeze(0).to(self.device)
-        image_data = base64.b64decode(satellite_image_b64)
-        image_stream = BytesIO(image_data)
-        with Image.open(image_stream) as img:
+        with Image.open(satellite_image) as img:
             z = transformation["satellite"](img).unsqueeze(0).to(self.device)
             height = img.height
             width = img.width
         self.model.eval()
         with torch.no_grad():
             cls_out, reg_out = self.model(x, z)  # используем только сls reg пустой
```

Ruff (0.15.12)

[warning] Unpacked variable `reg_out` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)